tooling/noir_js/test/node/e2e.test.ts

```diff
@@ -149,3 +151,24 @@ it('[BUG] -- bb.js null function or function signature mismatch (outer-inner) ',
   const isValidInner = await prover.verifyProof(_proofInner);
   expect(isValidInner).to.be.true;
 });
+
+it('end-to-end ACIR program proof creation and verification (inner)', async () => {
```

Comment:
Can you update the name so it's clear we're testing folding here?

Reply:
I changed it to `end-to-end proof creation and verification for multiple ACIR circuits`